astropy/units/quantity.py

```diff
@@ -607,6 +618,15 @@ def to(self, unit, equivalencies=[]):
         return self._new_view(new_val, unit)
 
     @property
+    def info(self):
```

Comment:
Yes, I actually did implement `@lazyproperty` across the board, but then realized that `lazyproperty` actually just sets the value in the object dictionary.  That means that `info` is not (AFAIK) an actual property and it's not read-only (so there is a chance to mess it up saying `col.info = 5`).

However, there was a more important and subtle problem which actually required going away from `lazyproperty` and now I don't remember the details.  Maybe it was related to copying columns on initialization.